libs/wingsdk/src/cloud/logger.ts

```diff
@@ -12,6 +12,11 @@ export const LOGGER_SYMBOL = Symbol.for(LOGGER_TYPE);
 export abstract class LoggerBase extends Resource {
   public readonly stateful = true;
 
+  constructor(scope: Construct, id: string) {
+    super(scope, id);
+    this.display.hidden = true;
```

Comment:
Maybe add a description as well?

Reply:
LOL that's not what I meant, I meant `this.display.description`

Reply:
Why a `display.description` of a Resource that will not be displayed on the Console?

Reply:
Mmmm. Good question... The way I see it, we should include a description for all the resources in the SDK, whether they are displayed or not. The reason:

1. The description can be used in other places, not just the console.
2. We might add support in the console to show hidden nodes at some point, in which case it makes sense to have a description for those nodes.

But that's not really critical... We can also add that later (and perhaps better to add it to all resources at once).

Reply:
Agree on both points 👍 

Let me add a title and a description to all resources on this PR.

Reply:
👍

Reply:
I added titles and descriptions from the [SDK ref](https://docs.winglang.io/reference/sdk).. Not the best ones I think.. :)